v2.8.0/language-es.yml

```diff
@@ -0,0 +1,300 @@
+# Language: es
+
+msg-shop-type-admin-regular: "Admin shop"
+msg-shop-type-selling: "Selling shop"
+msg-shop-type-buying: "Buying shop"
+msg-shop-type-trading: "Trading shop"
+msg-shop-type-book: "Book shop"
+
+msg-shop-type-desc-admin-regular: "tiene stock ilimitado"
+msg-shop-type-desc-selling: "vende items a los jugadores"
+msg-shop-type-desc-buying: "compra items de los jugadores"
+msg-shop-type-desc-trading: "tradea items con jugadores"
+msg-shop-type-desc-book: "vende copias de libro"
+
+msg-shop-object-type-living: "{type}"
+msg-shop-object-type-sign: "sign"
+msg-shop-object-type-npc: "npc"
+
+msg-selected-shop-type: "&aTipo de tienda seleccionada: &6{type} &7({description})"
+msg-selected-shop-object-type: "&aTipo de objeto seleccionado: &6{type}"
+
+msg-creation-item-selected: |
+    &aCreación de Tienda:
+    &e  Click izquierdo/derecho para seleccionar el tipo de tienda.
+    &e  Shift + click izquierdo/derecho para seleccionar el tipo de objeto.
+    &e  Click derecho en un cofre para seleccionarlo. 
+    &e  Click derecho en un bloque para spawnear al aldeano.
```

Comment:
"spawnear" is this correct? google translate doesn't seem to know this word.
Also, "aldeano": Keep in mind that there are other types of shopkeepers as well, besides villagers.. Maybe there is a more general word for 'shopkeeper'? Maybe simply 'shop' or 'trader' or similar? Or just 'shopkeeper'?